parsec/backend/invite_mail.tmpl.html

```diff
@@ -0,0 +1,441 @@
+            }
+            table[class=body] p,
+            table[class=body] ul,
+            table[class=body] ol,
+            table[class=body] td,
+            table[class=body] span,
+            table[class=body] a {
+            font-size: 16px !important; 
+            }
+            table[class=body] .wrapper,
+            table[class=body] .article {
+            padding: 10px !important; 
+            }
+            table[class=body] .content {
+            padding: 0 !important; 
+            }
+            table[class=body] .container {
+            padding: 0 !important;
+            width: 100% !important; 
+            }
+            table[class=body] .main {
+            border-left-width: 0 !important;
+            border-radius: 0 !important;
+            border-right-width: 0 !important; 
+            }
+            table[class=body] .btn table {
+            width: 100% !important; 
+            }
+            table[class=body] .btn a {
+            width: 100% !important; 
+            }
+            table[class=body] .img-responsive {
+            height: auto !important;
+            max-width: 100% !important;
+            width: auto !important; 
+            }
+        }
+
+        /* -------------------------------------
+            PRESERVE THESE STYLES IN THE HEAD
+        ------------------------------------- */
+        @media all {
+            .ExternalClass {
+            width: 100%; 
+            }
+            .ExternalClass,
+            .ExternalClass p,
+            .ExternalClass span,
+            .ExternalClass font,
+            .ExternalClass td,
+            .ExternalClass div {
+            line-height: 100%; 
+            }
+            .apple-link a {
+            color: inherit !important;
+            font-family: inherit !important;
+            font-size: inherit !important;
+            font-weight: inherit !important;
+            line-height: inherit !important;
+            text-decoration: none !important; 
+            }
+            #MessageViewBody a {
+            color: inherit;
+            text-decoration: none;
+            font-size: inherit;
+            font-family: inherit;
+            font-weight: inherit;
+            line-height: inherit;
+            }
+            .btn-primary table td:hover {
+            background-color: #34495e !important; 
+            }
+            .btn-primary a:hover {
+            background-color: #34495e !important;
+            border-color: #34495e !important; 
+            }
+        }
+        .aligncenter {
+            text-align: center;
+        }
+
+
+        </style>
+    </head>
+    <body class="">
+        <!-- These lines serve to isolate the preheader and avoid a preview of the mail's content -->
+        <span class="preheader">Follow these steps to join your new workspace
+            &nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;&nbsp;&zwnj;
```

Comment:
This is for the preheader text, I've done quite a bit of research and there doesn't seem to be an ideal solution for this issue. This prints whitespaces after the preheader text to avoid a preview of the email in the preheader, it should be clear on this screenshot
![Screenshot from 2020-06-10 11-40-24](https://user-images.githubusercontent.com/41511029/84252809-61175680-ab0f-11ea-85ed-3a02bc861f21.png)